services/audit-service/src/main/java/datawave/microservice/audit/replay/ReplayController.java

```diff
@@ -0,0 +1,726 @@
+        try {
+            replayTask = new ReplayTask(config, status, statusCache, replayProperties) {
+                @Override
+                protected boolean audit(Map<String,String> auditParamsMap) {
+                    return auditController.audit(msgHandlerAuditParams.fromMap(auditParamsMap));
+                }
+            };
+        } catch (Exception e) {
+            log.warn("Unable to create replay task for id " + status.getId(), e);
+        }
+        
+        if (replayTask != null) {
+            Future future = null;
+            try {
+                future = auditReplayExecutor.submit(replayTask);
+            } catch (TaskRejectedException e) {
+                log.warn("Unable to accept audit replay task for id " + status.getId(), e);
+            }
+            
+            if (future != null)
+                return new RunningReplay(status, future);
+        }
+        
+        return null;
+    }
+    
+    /**
+     * Starts all audit replays
+     *
+     * @return status, indicating the number of audit replays which were successfully started
+     */
+    @ApiOperation(value = "Starts all audit replays.")
+    @RolesAllowed({"Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/startAll", method = org.springframework.web.bind.annotation.RequestMethod.POST)
+    public String startAll() {
+        
+        log.info("Starting all audit replays");
+        
+        int replaysStarted = 0;
+        List<Status> statuses = statusCache.retrieveAll();
+        if (statuses != null) {
+            for (Status status : statuses) {
+                if (status.getState() == CREATED) {
+                    RunningReplay replay = start(status);
+                    if (replay != null) {
+                        runningReplays.put(status.getId(), replay);
+                        replaysStarted++;
+                    }
+                }
+            }
+        }
+        
+        log.info(replaysStarted + " audit replays started");
+        return replaysStarted + " audit replays started";
+    }
+    
+    /**
+     * Gets the status of an audit replay
+     *
+     * @param id
+     *            The audit replay id
+     * @return the status of the audit replay
+     */
+    @ApiOperation(value = "Gets the status of an audit replay.")
+    @RolesAllowed({"AuthorizedUser", "AuthorizedServer", "InternalUser", "Administrator", "JBossAdministrator"})
+    @RequestMapping(path = "/{id}/status", method = org.springframework.web.bind.annotation.RequestMethod.GET)
+    public Status status(@ApiParam("The audit replay id") @PathVariable("id") String id, HttpServletResponse response) throws IOException {
+        
+        log.info("Getting status for audit replay with id " + id);
+        
+        Status status = status(id, replayProperties.isPublishEvents());
+        
+        if (status == null)
+            response.sendError(HttpServletResponse.SC_BAD_REQUEST, "No audit replay found with id " + id);
```

Comment:
Same comment as previous about sending a bad request here instead of a not found.